DiscordChatExporter.Core.Rendering/Resources/HtmlCore.css

```diff
@@ -57,6 +57,70 @@ img {
     border-radius: 3px;
 }
 
+.spoiler.hidden {
+    cursor: pointer;
+}
+
+.spoiler.hidden .spoilerText {
+    opacity: 0;
+}
+
+.spoiler-image {
+    margin-top: 0.3em;
+    border-radius: 3px;
+    -webkit-box-shadow: 0.5px 0.5px 1px 1px rgba(0,0,0,.1);
+    box-shadow: 0.5px 0.5px 1px 1px rgba(0,0,0,.1);
+    cursor: pointer;
+    overflow: hidden;
+    position: relative;
+    width: fit-content;
+}
+
+.spoiler-image:hover .spoiler-warning {
+    color: #fff;
+    background-color: rgba(0,0,0,.9);
+}
+
+.spoiler-warning {
+    color: #dcddde;
```

Comment:
Are these colors the same across Dark and Light theme in Discord?

Reply:
Yes, they are.

Reply:
Great!